src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -38,29 +38,47 @@ namespace Orleans.TestingHost
     /// </remarks>
     public class TestingSiloHost
     {
+        /// <summary> Single instance of TestingSiloHost </summary>
         public static TestingSiloHost Instance { get; set; }
 
+        /// <summary> Primary silo handle </summary>
         public SiloHandle Primary { get; private set; }
+
+        /// <summary> List of handles to the secondary silos </summary>
         public SiloHandle Secondary { get; private set; }
-        protected readonly List<SiloHandle> additionalSilos = new List<SiloHandle>();
-        protected readonly Dictionary<string, GeneratedAssembly> additionalAssemblies = new Dictionary<string, GeneratedAssembly>();
 
-        protected TestingSiloOptions siloInitOptions { get; private set; }
-        protected TestingClientOptions clientInitOptions { get; private set; }
+        private readonly List<SiloHandle> additionalSilos = new List<SiloHandle>();
+        private readonly Dictionary<string, GeneratedAssembly> additionalAssemblies = new Dictionary<string, GeneratedAssembly>();
+
+        private TestingSiloOptions siloInitOptions { get; set; }
+
+        private TestingClientOptions clientInitOptions { get; set; }
+
+        /// <summary> Get or set the client configuration/// </summary>
         public ClientConfiguration ClientConfig { get; private set; }
+
+        /// <summary> Get or set the global configuration </summary>
         public GlobalConfiguration Globals { get; private set; }
 
+        /// <summary> The deploymentId value to use in the cluster </summary>
         public string DeploymentId = null;
+
+        /// <summary> The prefix to use in the deploymentId </summary>
         public string DeploymentIdPrefix = null;
 
+        /// <summary> Base port number for silos in cluster </summary>
         public const int BasePort = 22222;
+
+        /// <summary> Base port number for the gateway silos </summary>
         public const int ProxyBasePort = 40000;
 
+        /// <summary> Number of silos in the cluster </summary>
         private static int InstanceCounter = 0;
 
+        /// <summary> GrainFactory to use in the tests </summary>
         public IGrainFactory GrainFactory { get; private set; }
 
-        protected Logger logger
```

Comment:
I can revert this, but it is not used here? You mean that it could be used outside our codebase?